apps/web/modules/api/management/responses/types/responses.ts

```diff
@@ -0,0 +1,55 @@
+import { Contact, Response, ResponseNote, Tag, User } from "@prisma/client";
+import { z } from "zod";
+import { ZResponse } from "@formbricks/database/zod/responses";
+
+export type TResponseNew = Response & {
+  contact?: Pick<Contact, "id" | "userId">;
+  notes?: (Pick<ResponseNote, "id" | "text" | "createdAt" | "updatedAt" | "isEdited" | "isResolved"> & {
+    user: Pick<User, "id" | "name">;
+  })[];
+  tags: Tag[];
+};
+
+export const ZGetResponsesFilter = z.object({
+  surveyId: z.string().cuid2().optional(),
+  limit: z.coerce.number().default(10).optional(),
+  skip: z.coerce.number().default(0).optional(),
+  sortBy: z.enum(["createdAt", "updatedAt"]).default("createdAt").optional(),
+  order: z.enum(["asc", "desc"]).default("asc").optional(),
+  startDate: z.coerce.date().optional(),
```

Comment:
are these ISO 8601 strings in the query parameters since we cannot pass dates as query parameters

Reply:
Yes yes, the coerce function in the schema makes them date string